spec/cube_cache_control_spec.rb

```diff
@@ -86,22 +86,32 @@
       when 'mysql', 'jdbc_mysql', 'postgresql', 'oracle'
         @connection.execute 'CREATE TABLE sales_copy AS SELECT * FROM sales'
       when 'mssql', 'sqlserver'
-        @connection.execute 'SELECT * INTO sales_copy FROM sales'
+        @connection.raw_connection.execute 'SELECT * INTO sales_copy FROM sales'
```

Comment:
Found out that probably the cause is that in case of `mssql` the `_execute` method is redefined as

```ruby
def _execute(sql, name = nil)
  # Match the start of the SQL to determine appropriate behavior.
  # Be aware of multi-line SQL which might begin with 'create stored_pr
  # and contain 'insert into ...' lines.
  # NOTE: ignoring comment blocks prior to the first statement ?!
  if self.class.insert?(sql)
    if id_insert_table_name = identity_insert_table_name(sql)
      with_identity_insert_enabled(id_insert_table_name) do
        @connection.execute_insert(sql)
      end
    else
      @connection.execute_insert(sql)
    end
  elsif self.class.select?(sql)
    @connection.execute_query_raw repair_special_columns(sql)
  else # create | exec
    @connection.execute_update(sql)
  end
end
```

and `'SELECT * INTO sales_copy FROM sales'` is identified as a select query as it starts with `SELECT` and therefore `executeQuery` JDBC method is used. Probably a simple fix is to append a space in front of this SQL statement and use:

```ruby
@connection.execute ' SELECT * INTO sales_copy FROM sales'
```